File: CPlayer.java
class CPlayer implements IPlayer {
    //general client
    
    //data
    String color;
    //const
    CPlayer() { super(); }
    //functions
    public void startGame(String color) {
        this.color = color;
        System.out.println("My color is "+this.color);
    }
    
    public Move doMove(Board brd, int[] dice) {
        //implement
        return null;
    }
    
    public void DoublesPenalty() {
        //implement
    }
}

File: python/CPlayer.py
"""
Implements an automatic client player
"""

#Global

#Local
from Board import Board
from EnterPiece import EnterPiece
from MoveMain import MoveMain
from Player import Player
from RuleChecker import RuleChecker

class CPlayer(Player):
    def __init__(self, color, moves):
        Player.__init__(self)
        self.moves = moves
        self.color = color
        
    def doMove(self, board, dice):
        return self.moves
